src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -119,6 +131,24 @@ def load_command_table(self, _):
                                  setter_name='_firewall_rule_custom_setter', setter_type=rdbms_custom, setter_arg_name='parameters',
                                  custom_func_name='_firewall_rule_update_custom_func')
 
+    with self.command_group('mysql server vnet-rule', mysql_vnet_sdk) as g:
+        g.command('create', 'create_or_update')
+        g.command('delete', 'delete', confirmation=True)
+        g.command('show', 'get')
+        g.command('list', 'list_by_server')
+        g.generic_update_command('update',
+                                 getter_name='_custom_vnet_update_get', getter_type=rdbms_custom,
+                                 setter_name='_custom_vnet_update_set', setter_type=rdbms_custom, setter_arg_name='parameters')
+
+    with self.command_group('postgres server vnet-rule', postgres_vnet_sdk) as g:
+        g.command('create', 'create_or_update')
+        g.command('delete', 'delete', confirmation=True)
```

Comment:
Same as above.

Reply:
This can potential break the user connectivity to the server if rules are deleted by mistake , hence the additional check for confirm his/her intention.

Reply:
removed confirmation